meetings/2023-07/july-11.md

```diff
@@ -989,7 +1067,11 @@ NRO: Thank you, everyone.
 
 The proposal now uses the `import defer * as ns from "mod"` syntax to import modules without evaluating them, and the evaluation will happen synchronously when accessing properties on the namespace object. To ensure synchronous evaluation while still maintaining compatibility with top-level await, modules containing top-level await are still eagerly evaluated (together with their dependencies).
 
-The main points of the discussion were: How does this interact with Wasm modules? The current Wasm-ESM integration marks them as async. However, it may be possible to make their evaluation/instantiation synchronous and thus allow them to be deferred. So far the performance recommendation has been to "make things async" using dynamic import(). With this proposal, using `await` at the top-level of a module would prevent it from being deferred, going against that recommendation. How can we reconcile this? Forcing the namespace imports syntax may make it less statically analyzable for tools. However, it's likely that current heuristics (e.g., “it works as long as all accesses are . and not []”) are already good enough for this type of static analysis. Forcing the namespace syntax is also a compromise for ergonomics/orthogonality.
+The main points of the discussion were:
+
+- How does this interact with Wasm modules? The current Wasm-ESM integration marks them as async. However, it may be possible to make their evaluation/instantiation synchronous and thus allow them to be deferred.
```

Comment:
```suggestion
- How does this interact with WASM modules? The current WASM-ESM integration marks them as async. However, it may be possible to make their evaluation/instantiation synchronous and thus allow them to be deferred.
```